distributed/protocol/numpy.py

```diff
@@ -47,15 +47,17 @@ def serialize_numpy_ndarray(x):
               'strides': x.strides,
               'shape': x.shape}
 
+    data = x.view('u1').data
```

Comment:
I'm a little nervous about using view with a different dtype itemsize. See this warning from the docs:

> For a.view(some_dtype), if some_dtype has a different number of bytes per entry than the previous dtype (for example, converting a regular array to a structured array), then the behavior of the view cannot be predicted just from the superficial appearance of a (shown by print(a)). It also depends on exactly how a is stored in memory. Therefore if a is C-ordered versus fortran-ordered, versus defined as a slice or transpose, etc., the view may give different results.

It might be just fine for merely getting access to the `data` buffer, though.